app/views/shared/_breadcrumbs.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class='al-search-breadcrumb'>
   <%= link_to t('arclight.routes.home'), root_path %>
-  <%= t('arclight.breadcrumb_separator') %>
+  <span aria-hidden="true"><%= t('arclight.breadcrumb_separator') %></span>
```

Comment:
This partial could be a nice spot to reuse `#aria_hidden_breadcrumb_separator`.

Reply:
It looks like given the conflicting changes in PR #847, it might be better to leave these as is for now and reassess when resolving the merge conflict.